samza-kafka/src/main/scala/org/apache/samza/util/KafkaUtilJava.java

```diff
@@ -0,0 +1,12 @@
+package org.apache.samza.util;
+
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+
+
+public class KafkaUtilJava {
```

Comment:
Prefer inlining, or just calling this KafkaUtil, or if one already exists, moving method there (even if it's Scala).

Reply:
I figured that we need to be strict in avoiding new Scala code (or try to convert Scala code to Java as much as possible). It doesn't matter for this case though, since now this can just be kept within `KafkaConsumerProxy`. I'll move it back.